### ISSUE TITLE
Question: How to make all flows as maks

### ISSUE THREAD

**Issue by JuanruMaryGuo** [MouseLand/cellpose#1084]
Hi,

First, Cellpose is really a great project but I have a question for the codes:
```
masks, flows, styles, diams = model.eval(imgs, diameter=None, channels=channels, flow_threshold=0.5, cellprob_threshold=-6)
```
When I am using the basic models, I found the flows are giving the correct thing but masks is only returning half of the results. I tried to make `cellprob_threshold` extremely low but it it seems not changing much. Is there a way to return a mask with all the cells in flows?

Thank you in advance!

**Comment by JuanruMaryGuo** [comment]
Solved. Thanks!

**Comment by 123qltppx** [comment]
I had the same issue, could you please share how you solved it?
